### DISCUSSION TITLE
Button scaling problem when snoozing alerts

### DISCUSSION THREAD

**Discussion by user-hash** [NightscoutFoundation/xDrip#3522] [Bug reports] [answered]
![Screenshot_20240606_221700_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/55350273/a02820f8-e872-4fe7-ad2c-4a45fd55fe39)
![Screenshot_20240606_221653_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/55350273/271f0ad4-f430-4e1f-a5b4-19ea60de225c)
Hi,
when setting the snooze alert in latest Xdrip, the scaling of the buttons is not consistent. Sometimes the only the top button is visible , sometimes it works normaly. 
 The phone is S23

**Comment by Navid200** [comment, marked as answer]
This was fixed in the last Nightly.
Which xDrip release are you using?
You can see it if you go to the classic status page: https://navid200.github.io/xDrip/docs/xDrip-Version.html

**Comment by user-hash** [comment]
Thanks, closed as solved!
